src/Features/Core/Portable/Completion/CompletionHelper.cs

```diff
@@ -43,14 +43,13 @@ public static CompletionHelper GetHelper(Document document)
         /// results, or false if it should not be.
         /// </summary>
         public bool MatchesPattern(string text, string pattern, CultureInfo culture)
```

Comment:
>MatchesPattern [](start = 20, length = 14)

Likely, need to mark it as obsolete. Now it should be consumed by the old completion only.